src/test/java/com/automationexercise/tests/Tests/ContactUsFormTest.java

```diff
@@ -0,0 +1,42 @@
+package com.automationexercise.tests.Tests;
+
+import com.automationexercise.Pages.P02_ContactUsForm;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.automationexercise.DriverManager.DriverManager.getDriver;
+import static com.automationexercise.Utilities.DataUtils.getJsonData;
+
+public class ContactUsFormTest {
+    String fname;
+    String femail;
+    String fsubject;
+    String fmessage;
+    String path;
+    String url;
+
+    @BeforeClass
+    public void setUp() {
+        fname = getJsonData("data", "name");
+        femail = getJsonData("data", "email");
+        fsubject = getJsonData("data", "subject");
+        fmessage = getJsonData("data", "message");
+        path = "C:\\Users\\FAST\\IdeaProjects\\AutomationExercisePracticing\\src\\test\\resources\\test-data";
+        url = getJsonData("data","url");
```

Comment:
URL already existed on the environment.properties file you can get it using **getEnvironmentValue** function.